tensorzero-core/src/db/clickhouse/migration_manager/migrations/migration_0040.rs

```diff
@@ -0,0 +1,67 @@
+use super::check_column_exists;
+use crate::db::clickhouse::migration_manager::migration_trait::Migration;
+use crate::db::clickhouse::ClickHouseConnectionInfo;
+use crate::error::Error;
+use async_trait::async_trait;
+
+/// Adds an optional `name` column to datapoint tables for a human-readable label/name. This doesn't have to be
+/// unique across datapoints, and can show up in the UI.
+pub struct Migration0040<'a> {
+    pub clickhouse: &'a ClickHouseConnectionInfo,
+}
+
+const MIGRATION_ID: &str = "0040";
+
+#[async_trait]
+impl Migration for Migration0040<'_> {
+    async fn can_apply(&self) -> Result<(), Error> {
+        Ok(())
```

Comment:
this should check that the ChatInferenceDatapoint and JsonInferenceDatapoint tables exist